.circleci/config.yml

```
@@ -0,0 +1,64 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/ruby:2.6.3-node-browsers
+        environment:
+          PGHOST: localhost
+          PGUSER: myapp
+          RAILS_ENV: test
+      - image: postgres:9.6
+        environment:
+          POSTGRES_USER: myapp
+          POSTGRES_DB: myapp_test
+          POSTGRES_PASSWORD: ""
+
+    working_directory: ~/repo/taskmanager_app
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1.1-dependencies-{{ checksum "taskmanager_app/Gemfile.lock" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1.1-dependencies-
+
+      - run:
+          name: install dependencies
+          command:  bundle install --jobs=4 --retry=3 --path vendor/bundle
+          working_directory: taskmanager_app
+      - save_cache:
+          paths:
+            - taskmanager_app/vendor/bundle
+          key: v1.1-dependencies-{{ checksum "taskmanager_app/Gemfile.lock" }}
+
+      # Database setup
+      - run:
+          name: create Database
+          command: |
+            bundle exec rake db:create
+            bundle exec rake db:schema:load
+          working_directory: taskmanager_app
+
+      # run tests!
+      # Change to rspec if you are using rspec
+      - run:
+          name: run tests
+          command: |
+            bundle exec rspec spec/system/ \
```

```suggestion
            bundle exec rspec \
```
これで良いような気がします
systemディレクトリを指定したい理由(他のディレクトリのテストを動かしたくない理由)があればそのままままでも大丈夫です

---

現状特にディレクトリを絞る必要はないですね。変更します